zulip_bots/zulip_bots/bots/monkeytest/monkeytest.py

```diff
@@ -0,0 +1,41 @@
+from typing import Any
+
+import requests
+
+
+class MonkeyTestHandler(object):
+    def initialize(self, bot_handler: Any):
+        self.config_info = bot_handler.get_config_info('monkeytest')
+
+    def usage(self):
+        return '''
+        This is bot for Monkeytest.it(site for testing sites)
+        '''
+
+    def handle_message(self, message, bot_handler):
+        content = message['content'].strip()
+        if content == 'help' or content == '':
+            bot_handler.send_reply(message,
+                                   'This is bot for Monkeytest.it(site for testing sites)\nWrite me adress of site you want to test')
+        else:
+            if content.startswith('http://www.') or content.startswith('http://www.'):
+                url = content
+            else:
+                if content.startswith('www.'):
+                    url = 'http://' + content
+                else:
+                    url = 'http://www.' + content

```

Comment:
I'm not sure why you need to do this type of checking with URLs, plus there are also URLs that start with `https` and not just `http` where this would break. Could you please explain why you made these checks? :)